publish.sh

```diff
@@ -8,3 +18,9 @@ docker pull registry.tld/conjurinc/publish-rubygem
 summon --yaml "RUBYGEMS_API_KEY: !var rubygems/api-key" \
   docker run --rm --env-file @SUMMONENVFILE -v "$(pwd)":/opt/src \
   registry.tld/conjurinc/publish-rubygem parse_a_changelog
+
+docker tag parse-a-changelog "cyberark/parse-a-changelog:latest"
+docker tag parse-a-changelog "cyberark/parse-a-changelog:${TAG_NAME}"
+
+docker push "cyberark/parse-a-changelog:latest"
+docker push "cyberark/parse-a-changelog:${TAG_NAME}"
```

Comment:
up to you if you want to change, but in other scripts when a string is used multiple times we save it as a variable, so we could save "cyberark/parse-a-changelog" here (or even "cyberark" in one var, and "parse-a-changelog" in another)